NGPIris/hcp/hcp.py

```diff
@@ -216,30 +219,14 @@ def search_objects(self, string):
         return [obj for obj in self.objects if string in obj.key]
 
     @bucketcheck
-    def upload_file(self, local_path, remote_key, metadata={},callback="default"):
+    def upload_file(self, local_path, remote_key, metadata={}):
```

Comment:
Refering to our previous conversation.
Are you certain this works? Afaik pytests crash if `callback = ProgressPercentage(local_path)` is used.

Reply:
Removed left-over references to the callback kwarg and all tests pass